src/cli/help/env.go

```diff
@@ -46,11 +46,12 @@ func (ev envVar) MinimumPrintable() any {
 	return ev
 }
 
-func (ev envVar) Headers() []string {
+func (ev envVar) Headers(bool) []string {
+	// NOTE: skipID does not make sense in this context
```

Comment:
I don't understand this note as there's no reference to a skipID here, could you please clarify?